cloud-init-docker/cloud-init/apps/data-connect-trino/bootstrap-config/data-connect-trino-policy.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "data-connect-trino-access-policy",
+  "name": "Access Policy for Data Connect Trino",
+  "description": "Permissions required by data-connect-trino",
+  "statements": [
+    {
+      "principals": [
+        {
+          "type": "service",
+          "id": "${APP_NAME}-e2e-test"
+        }
+      ],
+      "actions": [
+        "data-connect:info",
+        "data-connect:data",
+        "data-connect:query"
+      ],
+      "resources": [
+        {
+          "uri": "${APP_DATA_CONNECT_TRINO_HOSTNAME}"
```

Comment:
I know it's a lot of work, but can we update all `APP_HOSTNAME` variables to `APP_APPNAME_URL`? Now that we are using the templated `externalUrl` or `resourceUrl` these values are no longer host names but full URLs.

Here's an example on the collection-service templates https://github.com/DNAstack/collection-service/blob/main/helm/templates/deployment.yaml#L96